services/update.py

```diff
@@ -243,27 +256,68 @@ def get_update(self):
 
         return False
 
-    def get_bad_rdf(self):
-        return bad_rdf
-
-    def get_rdf(self):
+    def get_output(self):
```

Comment:
Personally I would have split `get_output` into `get_json_output` and `get_rdf_output` and called the right one based on `use_json` down in `application` to avoid having to do `if self.use_json` at every level but it's not really worth refactoring now.

Reply:
I went back and forth with this, settled on this middle ground to make testing easier. I agree that if I were to do it all again I'd have fully separate methods... But hopefully in 3 months we can remove all the extra cruft anyway...